statler_api/models.py

```diff
@@ -52,6 +52,10 @@ class PlayListDAO(models.Model):
     title = models.CharField(max_length=256)
     url_title = models.CharField(max_length=32)
 
+    # See https://docs.djangoproject.com/en/dev/topics/db/models/#intermediary-manytomany
+    # for documentation on this many-to-many pattern.
+    plays = models.ManyToManyField(PlayDAO, through='PlayListEntryDAO')
```

Comment:
I understand that, but don’t know if there’s a use case for the plays in a list independent of their ranks.

Conceptually, that relationship does exist, so perhaps this mapping is worth including.